ksqldb-functional-tests/src/test/resources/query-validation-tests/avro.json

```diff
@@ -660,6 +660,33 @@
         "type": "io.confluent.ksql.util.KsqlStatementException",
         "message": "Avro only supports MAPs with STRING keys"
       }
+    },
+    {
+      "name": "should return error on field names incompatible with avro - C*",
+      "statements": [
+        "CREATE STREAM INPUT (1AvroDoesNotLikeFieldsThatStartWithNumbers DOUBLE) WITH (kafka_topic='input_topic', value_format='AVRO');"
```

Comment:
nit: this isn't a valid field name in ksqlDB (even Avro aside) so it'd be more realistic to put this in quotes.

Reply:
Good call. Thanks.